build/rpmbuild-docker/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN yum install -y --setopt=skip_missing_names_on_install=False \
 RUN yum -y remove nodesource-release-el7-1.noarch
 RUN yum clean all && rm -rf /var/cache/yum
 
-ENV GO_VERSION 1.14.4
+ENV GO_VERSION 1.14.8
 ENV GO_CHECKSUM aed845e4185a0b2a3c3d5e1d0a35491702c55889192bb9c30e67a3de6849c067
```

Comment:
Oops. @Sudokamikaze please update checksum :)

Reply:
Should I push into your branch or should I create another PR?

Reply:
Please update that PR and merge it once CI passes

Reply:
Actually, after manually confirming that it works – CI does not build rpmbuild